templates/instance.go

```diff
@@ -110,13 +110,25 @@ func Delete(w http.ResponseWriter, r *http.Request) {
 
 	var template *InstanceTemplate
 
+	templateAllocated, err := CheckTempplateAllocationByID(ctx, uuid, session.AccountID)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+	if templateAllocated {
```

Comment:
Error message: on second pass we use the work "use" twice - which sounds weird

Reply:
Indeed. Thank you!